custom_dc/load_data.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash
+# Script to process stats using simple stats loader.
```

Comment:
Why is this description different from the "USAGE" variable?

Reply:
Sorry, cut-paste  remnant. fixed.